### PR TITLE
GEN-2159: Added new fonts sizes

### PR DESCRIPTION
**Pull request by sladan-hedvig**
## [APP-XXX]

- Added font sizes
- Replaced some of them
- Will remove old ones when we update all of the views

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

**Review by julia-andersson — APPROVED** [review]

**Comment by sladan-hedvig** [comment]
Will merge after we get approval for 12.6.3

**Pull request merged**
